Documentation/operations/upgrade.rst

```diff
@@ -340,6 +340,9 @@ Helm Options
   were deprecated in Cilium 1.14 in favor of ``tls.ca.cert`` and ``tls.ca.key`` respectively,
   and have been removed. The ```clustermesh-apiserver-ca-cert`` secret is no longer generated.
 
+* Values ``authentication.mutual.spire.install.agent.image`` and ``authentication.mutual.spire.install.server.image``
+  in Cilium 1.14 is changed to decouple the repository and tag, which could better deploy in an offline environment.
```

Comment:
these release notes already target v1.15 of cilium - and no longer 1.14.

I wouldn't name the version at all - because the version should already be clear from the context. I would rather focus on describing the actual type-change of the values

-> `Values ``authentication.mutual.spire.install.agent.image`` and ``authentication.mutual.spire.install.server.image``
  changed their type from a string to a structured definition that decouples repository and tag. This improves the usage in offline environments.`

@zacharysarah wdyt?

Reply:
agree, and change to your suggestion

Reply:
@weizhoublue Thanks - looks like you missed the last dot in the sentence.

Reply:
thanks